documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,405 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere, {rhv-full}, or {osp} to {virt} with {the-lc} {project-first}.
```

Comment:
this part seems to go back to MTV 2.4 where we had only vSphere, RHV and OpenStack as source providers - we should rather that section from the release notes of 2.5 that include also OVA and OpenShift

Reply:
changed